### PR TITLE
Adds misspell-fixer-action

### PR DESCRIPTION
**Pull request by sobolevn**
I have also added a new section for "Localization" from the marketplace: https://github.com/marketplace/category/localization

Link: https://github.com/sobolevn/misspell-fixer-action
Marketplace: https://github.com/marketplace/actions/misspell-fixer-action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,6 +427,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [Setup Cassandra Schema](https://github.com/fabasoad/setup-cassandra-action) - Running scripts from the provided folder on top of Cassandra cluster.
 
+### Localiation
```
Comment (papb) [inline README.md]:
```suggestion
### Localization
```

Typo :sweat_smile:

Comment (sobolevn) [reply README.md]: Ahhahahah 😆

**Comment by sobolevn** [comment]
Well, one action says that there are a lot of broken links. And they seem to be alive.

**Comment by fabasoad** [comment]
I have the same Lint failure in my PR https://github.com/sdras/awesome-actions/pull/352 😢 I have no idea what is it. Waiting for the owner's comment...

File: README.md [hunk 2]
```diff
@@ -426,6 +427,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [Setup Cassandra Schema](https://github.com/fabasoad/setup-cassandra-action) - Running scripts from the provided folder on top of Cassandra cluster.
 
+### Localization
+
+- [Find and automatically fix typos and grammar issues in your code](https://github.com/sobolevn/misspell-fixer-action)
```
Comment (jessicalostinspace) [inline README.md]: I'm a little hesitant to add this to its own category. My reasoning is that the action in question here not only localizes, but fixes the typos and issues as well, which I would consider linting. Any thoughts on that?

Comment (jessicalostinspace) [reply README.md]: Also, lint dead-url check is temporarily disabled, so you will have to pull and push again.

Comment (sobolevn) [reply README.md]: Yes, I agree with you hesitation. My thought was: linters are more like code oriented. When localization is more NL oriented. And that's why I made this decision. 

I guess it really belongs there.

**Pull request merged**